backend/src/expressServer.ts

```diff
@@ -77,6 +78,7 @@ export const startExpressServer = async () => {
     privateRoutesRouter.use(requireAuth())
 
     app.get('/api/health', health)
+    app.use('/api/auth', authRateLimiter, googleAuthRouter)
```

Comment:
nit: is /auth gonna handle other things beyond google? cause now /auth goes to googleAuthRouter.

we should either call this oAuthRouter if it will handle all oauth (github etc) or /google_auth if it's just gonna be google?